### ISSUE TITLE
zero improvement for the paraformer training recipe.

### ISSUE THREAD

**Issue by JayC319** [alibaba-damo-academy/FunASR#920]
Hi I am following the simple flow of run.sh of the paraformer structure on AISHELL1, which is the example flow ,and I am in stage 4. However, from the very start of the 1st epoch, there were never improvement of the model. The only adjustment I made was changing the batch size from 25000 to 10000 and GPU num into 1 in train_asr_paraformer_conformer_12e_6d_2048_256.yaml. Wonder where did I go wrong.

The result always stuck at the following number:
`[train] iter_time=2.106e-04, forward_time=0.111, loss_ctc=0.030, loss_att=4.551e-04, pre_loss_att=4.577e-04, acc=1.000, loss_pre=1.110e-05, loss=0.010, backward_time=0.120, optim_step_time=0.039, optim0_lr0=1.158e-04, train_time=0.339, time=29 minutes and 57.69 seconds, total_count=562436, gpu_max_cached_mem_GB=8.520, [valid] loss_ctc=234.760, cer_ctc=0.922, loss_att=34.787, pre_loss_att=34.789, acc=0.781, cer=0.266, wer=0.958, loss_pre=15.827, loss=134.959, time=1 minute and 4.73 seconds, total_count=67522, gpu_max_cached_mem_GB=8.520`

**Comment by hnluo** [comment]
It seems some problems with the training data, did you run the training recipe from stage 0 to stage 4.

**Comment by JayC319** [comment]
Yes, I download the AISHELL dataset and make sure the scp file, wav file and the fbanks jsons are all there. Should I refetchthe package all over again?

Since this is my second trial of running run.sh

**Comment by hnluo** [comment]
You can check training data directory, whether total lines of wav.scp and shape files is 120098，and whether there are spaces between Chinese characters.

**Comment by JayC319** [comment]
Sorry for the late reply, the lines of wav.scp and shape files are all 120098 + additional space line.  
The speech in text is {ID} {content without space} , In text.org, the text is {ID} {content with space}.
